src/software/simulation/physics/physics_world_test.cpp

```diff
@@ -13,11 +15,14 @@ class PhysicsWorldTest : public ::testing::Test
     {
         simulator_config = std::make_shared<const SimulatorConfig>();
         physics_world = std::make_shared<PhysicsWorld>(Field::createSSLDivisionBField(),
+                                                       robot_constants, wheel_constants,
                                                        simulator_config);
     }
 
     std::shared_ptr<const SimulatorConfig> simulator_config;
     std::shared_ptr<PhysicsWorld> physics_world;
+    RobotConstants_t robot_constants = create2015RobotConstants();
+    WheelConstants_t wheel_constants = create2015WheelConstants();
```

Comment:
just thinking, is the idea that eventually we'll be able to remove the direct dependency of most things on `2015_robot_constants` and can instead just depend on some centralized interface for getting them?

Reply:
no we want to keep direct dependency so we can change simulation vs software vs firmware independently